src/ChainSafe.Gaming.Marketplace/MarketplaceClient.cs

```diff
@@ -70,15 +57,13 @@ public MarketplaceClient(IHttpClient httpClient, IProjectConfig projectConfig, I
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
         public Task<MarketplacePage> LoadPage(
             MarketplacePage? currentPage = null,
-            string? marketplaceId = null,
```

Comment:
Do we need to force user into using just one marketplace in his game?

Reply:
For now I'd say so.
I mean _techcnically_ , we should be able to easily add the ability for users to load multiple marketplaces, but let's do it post 3.0. I will remove the marketplaceId from there since it doesn't make sense to be in the LoadPage